crates/ruff_linter/src/rules/pyflakes/rules/unused_import.rs

```diff
@@ -90,7 +95,7 @@ impl Violation for UnusedImport {
             }
             Some(UnusedImportContext::Init) => {
                 format!(
-                    "`{name}` imported but unused; consider adding to `__all__` or using a redundant alias"
+                    "`{name}` imported but unused; consider removing, adding to `__all__`, or using a redundant alias"
```

Comment:
I'd usually refer to this as an "explicit reexport" rather than "redundant alias", FWIW, but "redundant alias" is probably clearer for people unfamiliar with the idea